CHANGELOG.md

```diff
@@ -9,6 +9,8 @@
 * New `mergeDeep` method provided in `@xh/hoist/utils/js` as an alternative to `lodash.merge`,
   without lodash's surprising deep-merging of array-based properties.
 * Enhanced Roles Admin UI to support bulk category reassignment.
+* New `Empty Value` prop for select inputs, allowing for a custom value to be returned when select
```

Comment:
Suggest:

> Added new `select.emptyValue` prop, allowing a custom value to be....

In general for things like new props and configs, best to use the exact name of the prop